src/script/buildcontainer-setup.sh

```diff
@@ -36,7 +36,8 @@ case "${CEPH_BASE_BRANCH}~${DISTRO_KIND}" in
         install_container_deps
         dnf_clean
     ;;
-    *~*centos*9|*~*centos*10*|*~fedora*)
+    # EL-ish, 9+
+    *~*centos*9|*~*centos*10*|*~fedora*|*~rocky*|*~alma*)
```

Comment:
since we have a centos8 case above:

```suggestion
    *~*centos*|*~fedora*|*~rocky*|*~alma*)
```

Reply:
Fixed over in #64072 and inherited here via rebase. PTAL at #64072 :-)